.changeset/cuddly-turkeys-love.md

```diff
@@ -0,0 +1,12 @@
+---
+"wrangler": patch
+---
+
+fix: don't crash cli on older versions of node
+
+The entry point for the cli `./bin/wrangler.js` wasn't being transpiled, which meant that any newer syntax would cause it to crash on older versions of node. This is a problem because we need this script to run even on unsupported versions so we can log a message telling the user their version of node is too old. The fix here is to just transpile the entry point as well.
+
+Additionally, this fix also runs `emit-types` only when doing a full build, so local dev is a little faster.
```

Comment:
Feels like this should be two changesets since the two changes are completely unrelated?

Reply:
done